builder_test.go

```diff
@@ -372,17 +372,17 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 				if s := cmp.Diff(metadata, &lifecycle.BuildMetadata{
 					Processes: []launch.Process{},
 					Buildpacks: []lifecycle.Buildpack{
-						{ID: "A", Version: "v1", API: "0.3"},
+						{ID: "A", Version: "v1", API: "0.3", Homepage: "Buildpack A Homepage"},
 						{ID: "B", Version: "v2", API: "0.2"},
 					},
 					BOM: []lifecycle.BOMEntry{
 						{
 							Require:   lifecycle.Require{Name: "dep1", Version: "v1"},
-							Buildpack: lifecycle.Buildpack{ID: "A", Version: "v1"},
+							Buildpack: lifecycle.Buildpack{ID: "A", Version: "v1", Homepage: "Buildpack A Homepage"},
```

Comment:
I think it is okay to have the `homepage` in the `buidpacks` section of `metadata.toml`, but I think we should leave it out of the BOM entries like we do with API https://github.com/buildpacks/lifecycle/blob/main/builder.go#L207

Reply:
I'm surprised that we would want to leave the homepage out of the BOM, but I also don't know how the BOM will be used or displayed to users.

Reply:
I think given that the image metadata now contains the homepage for each buildpack we don't need to repeat it in each BOM entry. The BOM entry only needs enough data to uniquely identify the buildpack. The homepage can be looked up using the buildpack `id` and `version`.